src/components/rtc-settings-panel.tsx

```diff
@@ -75,6 +88,14 @@ export function RTCSettingsPanel() {
 		setSelfAwarenessEnabled( enabled );
 	};
 
+	const handleToggleNotificationsForJoin = ( enabled: boolean ) => {
+		setNotificationsForJoinEnabled( enabled );
```

Comment:
My only concern with this approach is that, could we have too many settings to manage as we add more notifications? We are going to have two more toasts to manage so that'll be 4 notification settings to manage. I don't have a better solution that doesn't just do this exact same thing in a different manner.